surefire-providers/surefire-junit47/src/main/java/org/apache/maven/surefire/junitcore/FilteringRequest.java

```diff
@@ -32,7 +32,7 @@
 {
     private Runner filteredRunner;
 
-    public FilteringRequest( Request req, Filter filter )
+    FilteringRequest( Request req, Filter filter )
```

Comment:
why removing public constructor this is not backward compat???? if checkstyle complains about this so remove checkstyle usage as it's useless....

Reply:
This constructor wasn't used outside the package. Intellij also confirms it, so checkstyle reports it properly. Also keep in mind that tests passes. If it not being used, access can be safly lowered to package level.